release-lint/action.yaml

```diff
@@ -0,0 +1,163 @@
+name: GitHub Action Release Notes Preview
+description: GitHub Action that publishes a new release.
+inputs:
+  github-token:
+    required: true
+    description: GitHub token that can checkout the repository as well as create tags/releases against it. e.g. 'secrets.GITHUB_TOKEN'
```

Comment:
does the token really need to have permissions to create tags/releases?

Reply:
Not sure about that, I just reused the description from other actions to be consistent